tfjs-converter/python/tensorflowjs/converters/keras_tfjs_loader.py

```diff
@@ -92,6 +98,62 @@ def _deserialize_keras_model(model_topology_json,
 
   return model
 
+def _deserialize_keras_keras_model(model_topology_json,
```

Comment:
The name `_deserialize_keras_keras_model` includes `keras` twice. What does this mean? Should it be `_deserialize_keras_v3_model`?

Reply:
Oh, I just saw Lin's comment about this.